test/unit/failOnError.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const assert = require('assert');
+
+const sharp = require('../../');
+const fixtures = require('../fixtures');
+
+describe('failOnError', function () {
+  it('handles truncated JPEG by default', function (done) {
+    sharp(fixtures.inputJpgTruncated)
+      .resize(320, 240)
+      // .toFile(fixtures.expected('truncated.jpg'), done);
```

Comment:
those comments were left in so Engineers of Tomorrow can easily re-create the truncated images (in case VIPS rendering of truncated images changes). 

Do you still want me to remove them?

Reply:
OK, let's leave them in.